debian/rules

```diff
@@ -22,6 +22,19 @@ KERNEL_TYPE.esx := generic
 KERNEL_TYPE.gcp := gcp
 KERNEL_TYPE.kvm := kvm
 
+#
+# The DELPHIX_SIGNATURE_VERSION field refers to the current Delphix
+# Release version being built and only defaults to 6.0 if not set.
+# This is passed as an environment variable via jenkins job(s) that
+# launch the linux-pkg build for this package.
+#
+DELPHIX_SIGNATURE_VERSION := $(or $(DELPHIX_SIGNATURE_VERSION),6.0)
```

Comment:
`DELPHIX_SIGNATURE_VERSION` could be missing when :
1)  We push these changes first (without the devops-gate changes) or
2) Building this directly on a bootstrap VM (like `./scripts/docker-run.sh make packages` or via linux-pkg build like `./buildpkg.sh delphix-platform`) for testing etc.

Other than that I don't know of any other workflows where this could be missing.

Reply:
In that case, it might be safer to require `DELPHIX_SIGNATURE_VERSION` to be defined here and wait for the devops change before pushing this change. The `6.0` default might cause headaches down the road.

Reply:
I agree with @rasantel; I thought that was already the plan (to push the devops change first), but perhaps not.

For case (2), that's already broken, so I wouldn't worry about it; see here: https://jira.delphix.com/browse/TOOL-8270

Reply:
Fixed.